docs/maintainers/issue-management.md

```diff
@@ -74,6 +75,7 @@ The following labels are used on PRs:
 * `Review - Abandoned` : The PR was not updated for significant number of days (the exact number could vary over time).
   Maintainers should look into such PRs and re-evaluate them.
 * `Review - Committee` : The PR/Issue needs a review from [powershell-committee](../community/governance.md#powershell-committee)
+* `Review - Need Documentation` : The PR has changes that require a documentation change or new documentation added to [PowerShell-Docs](http://github.com/powershell/powershell-docs)
```

Comment:
Intent is to label PRs that have doc implications, but we agreed we wouldn't block accepting PRs due to lack of doc.  I put it with the `Review` group of labels as it seemed appropriate to get attention.  I'm fine with standalone `Documentation Needed`